packages/react-native-bpk-component-touchable-overlay/stories.js

```diff
@@ -66,4 +66,40 @@ storiesOf('react-native-bpk-component-touchable-overlay', module)
     >
       {viewContent}
     </BpkTouchableOverlay>
+  ))
+  .add('Examples', () => (
+    <View>
+      <BpkText>Example with Border Radius sm</BpkText>
```

Comment:
```suggestion
      <BpkText>With borderRadius="sm"</BpkText>
```

..and so on for the below examples, just to make it really clear what we're doing in each one.